app.js

```diff
@@ -28,6 +32,10 @@ app.use((err, req, res, next)=> {
     }
 })
 
+app.use((err, req, res, next) => {
+    console.log(err)
+})
```

Comment:
Dont forget to remove console.logs and things that were used for debugging. Code we push up to github should be complete and without any other unnecessary code.